components/eamxx/src/physics/mam/eamxx_mam_dry_deposition_process_interface.cpp

```diff
@@ -405,22 +428,21 @@ void MAMDryDep::run_impl(const double dt) {
   // Surface deposition flux of interstitial aerosols, [kg/m2/s] or [1/m2/s]
   auto aerdepdryis_ = get_field_out("deposition_flux_of_interstitial_aerosols")
                           .get_view<Real **>();
-
-  // FIXME: remove it if it read from a file
-  populated_fraction_landuse(fraction_landuse_, ncol_);
-
+  printf("BALLI%0.15e\n:", frac_landuse_(0, 1));
```

Comment:
This is the reason for the fails: you are accessing a dev view from host. I'm assuming this was a print statement for debug only, so it can be removed?